models/pathology_tumor_detection/configs/metadata.json

```diff
@@ -1,7 +1,8 @@
 {
     "schema": "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_20220324.json",
-    "version": "0.4.7",
+    "version": "0.4.8",
     "changelog": {
+        "0.4.8": "update the readme file with TensorRT convert",
         "0.4.7": "add name tag",
         "0.4.6": "modify dataset key name",
         "0.4.5": "update model weights and perfomance metrics",
```

Comment:
could `torch_tensorrt` version be added into "optional_packages_version" below? Thanks

Reply:
Hi @binliunls , it seems `pipenv` cannot find torch tensorrt 1.3.0  I rethought this situation and now I think we can remove it in metadata (we use MONAI docker for tensorrt related tests thus it doesn't matter here)
Could you please help to:
1. remove "torch_tensorrt"
2. modify "monai_version" into `1.2.0rc2`

Reply:
Hi @yiheng-wang-nv ,

About the MONAI verison in metadata, I am OK with "1.2.0rc2" or "1.2.0", it's only on `dev-1.2` branch.
Which one do you prefer? I would suggest "1.2.0" because we will upload bundles to NGC for QA testing before MONAI 1.2 release, if no bugs, then we don't need to change anything before really publish the bundles on NGC.

Thanks.

Reply:
Hi Nic, `1.2.0` will not pass ci test in model zoo, thus I suggested to use `1.2.0rc2`. I think we should use correct version here.

Reply:
Otherwise, let me add a workaround to test in dev-1.2, and remove it before merging into dev